tests/config/test_user_config.py

```
@@ -6,198 +6,273 @@
+        assert config.model_dump_json(exclude_unset=True, exclude_defaults=True) != config.model_dump_json()  # fmt: skip
+        assert config.model_dump_json(exclude_none=True) != config.model_dump_json()  # fmt: skip
+
+    @pytest.mark.parametrize(
+        "model_names,endpoint_type,timing_mode,streaming,expected_dir",
+        [
+            (
+                ["hf/model"],  # model name with slash
+                EndpointType.CHAT,
+                TimingMode.REQUEST_RATE,
+                True,
+                "/tmp/artifacts/hf_model-openai-chat-concurrency5-request_rate10.0",
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/artifacts/hf_model-openai-chat-concurrency5-request_rate10.0"

(S108)